azext_edge/edge/providers/rpsaas/adr/params.py

```diff
@@ -599,3 +599,41 @@ def load_adr_arguments(self, _):
             arg_group="Authentication",
             deprecate_info=context.deprecate(hide=True)
         )
+    # ADR REFRESH
+    with self.argument_context("iot ops namespace") as context:
+        context.argument(
+            "namespace_name",
+            options_list=["--name", "-n"],
+            help="Namespace name.",
```

Comment:
nit: as user I probably want to know more about what is namespace, for example namespace contained a collection of devices and assets, something like that?

Reply:
oh ho ho 
![image](https://github.com/user-attachments/assets/c16516c9-ef47-4b30-a170-3c9e2a1af6f3)

I had this already